blockchain/utxocache_test.go

```diff
@@ -413,13 +413,10 @@ func TestUtxoCache_Reorg(t *testing.T) {
 	// Spend all spendable outs and confirm with a block.
 	var forkSpendables []*spendableOut
 	b1a, forkSpendables := addBlock(chain, b0, spendableOuts)
-	b2a, forkSpendables := addBlock(chain, b1a, forkSpendables)
+	_, forkSpendables = addBlock(chain, b1a, forkSpendables)
 
-	if err := chain.FlushCachedState(FlushRequired); err != nil {
-		t.Fatalf("unexpected error while flushing cache: %v", err)
-	}
-	assertConsistencyState(t, chain, ucsConsistent, b2a.Hash())
-	assertNbEntriesOnDisk(t, chain, len(forkSpendables))
+	// leave the spent outs in the cache which lets us exercise
+	// un-spending entries both in the db and in the utxocache
```

Comment:
Actually... I'm not sure there are any spent in the DB at this point.

Reply:
yeah I don't think there are any in the db since it hasn't been flushed.